tests/analysis/rdf/test_rdf.py

```diff
@@ -1,16 +1,27 @@
 """Pytest for dynsight.analysis.compute_rdf."""
 
+import os
 from pathlib import Path
+from typing import Generator
 
-import MDAnalysis
 import numpy as np
 import pytest
 
-from dynsight.analysis import compute_rdf
+from dynsight.trajectory import Trj
 
 from .case_data import RDFCaseData
 
 
+@pytest.fixture
+def original_wd() -> Generator[Path, None, None]:
```

Comment:
when and how is this used? I do not see why you need to change dir?


Is it possible instead to do:

```python
original_dir = ...
try:
    do test
finally:
    chdir back to what you want
```

Reply:
OHhh it is something to do with the temp file, hey? If so, leave it - but this is confusing code, in my opinion.

Reply:
There must be a better way, but for now, I do not know. Writing an issue for future